src/main/java/ch/njol/skript/SkriptCommand.java

```diff
@@ -332,6 +333,14 @@ public boolean onCommand(final @Nullable CommandSender sender, final @Nullable C
 			} else if (args[0].equalsIgnoreCase("info")) {
 				info(sender, "info.aliases");
 				info(sender, "info.documentation");
+				info(sender, "info.server", Bukkit.getVersion());
+				info(sender, "info.version", Skript.getVersion());
+				info(sender, "info.addons");
+				for (SkriptAddon addon : Skript.getAddons()) {
+					PluginDescriptionFile desc = addon.plugin.getDescription();
+					String web = desc.getWebsite();
+					Skript.info(sender, " - " + desc.getFullName() + (web != null ? " (" + web + ")" : ""));

```

Comment:
this could also be in the lang file and you could insert the data with String#format

Reply:
I considered that, but then I thought a few things:
1) Is it really worth doing a line in the lang file for a hyphen and parenthesis?
2) would I have to do 2 lines in the lang file? 1 for no parenthesis, 1 for with parenthesis?

At the end of the day, who is really going to want to change a line that just contains a hyphen and a couple parenthesis?
Do we really think that's worth it?

Reply:
i guess not but what do you think of adding the author of the addon aswell